packages/fx-core/resource/package.nls.json

```diff
@@ -852,6 +852,7 @@
   "driver.apiKey.error.clientSecretInvalid": "Client secret is invalid. The length of client secret should be in this range: >=10 and <=128",
   "driver.apiKey.error.domainInvalid": "Domain is invalid. Domain for API key should follow: 1. Max %d domain(s) per API key. 2. Use comma to separate domains",
   "driver.apiKey.error.failedToGetDomain": "Unable to get domain from API specification. Make sure your API specification is valid.",
+  "driver.apiKey.error.clientSecretFromScratchInvalid": "Client secret is invalid. If you are working on an new Open API spec, please refer to the ReadMe for more detail",
```

Comment:
Proposed: Invalid client secret. If you are working on an new Open API spec, refer the ReadMe file for details.

Reply:
Updated, thanks!

Reply:
`If you are working on an new Open API spec` What does this mean?

Reply:
@MuyangAmigo I mean the from scratch template.

Reply:
@KennethBWSong Then can we say: `If you start with a new API, refer to the README file for details.`

Reply:
@MuyangAmigo  @supkasar Updated, please help to review, thanks!